index.html

```diff
@@ -170,7 +170,7 @@ <h4>Code meets data</h4>
                     <div class="col-md-6 nb-highlight-text">
                         <h3>Communicate your results with Voilà.</h3>
                         <h4 class="nb-desc">
-                            Voilà helps you communicate insights, by transforming a Jupyter Notebook into a stand-alone web application you can share. It gives you control over what your readers experience in a secure and customizable interactive dashboard.
+                            Voilà helps communicate insights by transforming notebooks into a secure, stand-alone web application that you can customize and share.
```

Comment:
I think this language is snappier, thanks!

I would suggest that "notebooks" and "web applications" should match in number.

```suggestion
                            Voilà helps communicate insights by transforming notebooks into secure, stand-alone web applications that you can customize and share.
```

Reply:
Thanks, I accepted your proposal.